File: Code/Disney/disney.xBandController/src/windows/Test/Disney.xBand.IDMS/Parties/Party.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;
using Disney.xBand.IDMS.Links;

namespace Disney.xBand.IDMS.Parties
{
    [DataContract]
    public class Party
    {
        [DataMember(Name = "partyId")]
        public long PartyID { get; set; }

        [DataMember(Name = "primaryGuestId")]
        public long PrimaryGuestID { get; set; }

        [DataMember(Name = "partyName")]
        public String PartyName { get; set; }

        [DataMember(Name = "count")]
        public int Count { get; set; }

        [DataMember(Name = "members")]
        private List<PartyMember> Members { get; set; }

        [DataMember(Name = "links")]
        private LinkCollection Links { get; set; }
    }
}

File: Code/Disney/disney.xBandController/src/linux/IDMSLib/src/main/java/com/disney/xband/idms/lib/model/Party.java
package com.disney.xband.idms.lib.model;

import java.util.List;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class Party 
{
	private long partyId;
    private long primaryGuestId;
	private String partyName;
	private int count;
	private List<PartyMember> members;
	private LinkCollection links;
	
	@XmlElement(name="partyId")
	public long getPartyId()
	{
		return this.partyId;
	}
	
	public void setPartyId(long partyId)
	{
		this.partyId = partyId;
	}
	
	@XmlElement(name="primaryGuestId")
	public long getPrimaryGuestId() 
	{
		return primaryGuestId;
	}
	
	public void setPrimaryGuestId(long primaryGuestId) 
	{
		this.primaryGuestId = primaryGuestId;
	}
	
	@XmlElement(name="count")
	public int getCount() 
	{
		return count;
	}
	
	public void setCount(int count) 
	{
		this.count = count;
	}
	
	@XmlElement(name="members")
	public List<PartyMember> getMembers() 
	{
		return members;
	}
	
	public void setMembers(List<PartyMember> members)
	{
		this.members = members;
	}
	
	@XmlElement(name="links")
	public LinkCollection getLinks() 
	{
		return links;
	}
	
	public void setLinks(LinkCollection links) 
	{
		this.links = links;
	}
	
	@XmlElement(name="partyName")
	public String getPartyName()
	{
		return this.partyName;
	}
	
	public void setPartyName(String partyName)
	{
		this.partyName = partyName;
	}
}
